Core/include/Acts/Propagator/Propagator.hpp

```diff
@@ -394,6 +428,38 @@ class Propagator final {
   propagate(const parameters_t& start, const Surface& target,
             const propagator_options_t& options) const;
 
+  /// @brief Propagate track parameters - User method
+  ///
+  /// This function performs the propagation of the track parameters according
+  /// to the internal implementation object until at least one abort condition
+  /// is fulfilled, the destination surface is hit or the maximum number of
+  /// steps/path length as given in the propagation options is reached.
+  ///
+  /// @tparam parameters_t Type of initial track parameters to propagate
+  /// @tparam surface_t Type of target surface
```

Comment:
This documentation is slightly incorrect I think, the template parameters `target_aborter_t` and `path_aborter_t` are not documented, but therefore nonexisting template parameters.

Reply:
Ok, this applies to all overloads i think.